README.md

```diff
@@ -101,6 +101,27 @@ All of the above still applies, but note:
 > [!NOTE]
 > Spaces are important here
 
+#### Ollama
```

Comment:
I wouldn't put that here, I'd create a specific adaptors section describing the feature with an example like the one you provided if that makes sense.